bot/tests/administration/test_add_subscription.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from bot.tests.base_test import BaseTest
+
+
+class TestAddSubscriptionCommand(BaseTest):
+
+    @pytest.mark.quick
+    def test_add_subscription_valid_user(self):
+        response = self.send_command('/addsub 2015344951 30')
+        expected_fragments = ["✅ Subskrypcja dla użytkownika 2015344951"]
```

Comment:
staraj sie wyciagac powtarzajace sie rzeczy do zmiennych, bo jak w tym losowym numerku sie trzepniesz to bedziesz 5min idiotycznego bledu szukac

Reply:
to też dotyczy wszystkiego